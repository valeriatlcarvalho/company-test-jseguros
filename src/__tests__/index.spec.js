import Index from '../index'

it('Renders', () => {
  expect(JSON.stringify(
      Object.assign({}, Index, { _reactInternalInstance: 'censored' }),
    )
  ).toMatchSnapshot()
})