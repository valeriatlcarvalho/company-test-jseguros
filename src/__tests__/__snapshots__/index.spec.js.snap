// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders 1`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;
